### PR TITLE
Update the content to the service description page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the content throughout the service description page. Fixed the use of the back-to-top pattern.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that is matches the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#)

### CODE REVIEW

**Comment by cjohnston1158** [comment]
Please make the following changes:
- Remove Cloud Region from ToC
- Remove Docker from ToC
- Add Livepatch on Prem to ToC
- Add Rancher to ToC

**Comment by anthonydillon** [comment]
@cjohnston1158 thanks, done.

**Comment by deadlight** [comment]
@anthonydillon we removed the blocked UX-needed label (it can be checked after release). Fix the conflict and you're good to go :+1:

File: templates/legal/terms-and-policies/privacy-policy/third-parties.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1>Third-party privacy policies</h1>
   <div class="row">
     <ul class="inline-list">
       <li class="p-list__item"><a href="http://www.3sat.de/page/?source=/service/140014/index.html">3sat Mediathek</a></li>
-      <li class="p-list__item"><a href="http://about.7digital.com/terms-and-conditions-use">7Digital</a></li>
+      <li class="p-list__item"><a href="http:/about.7digital.com/terms-and-conditions-use">7Digital</a></li>
```
Comment (Caleb-Ellis) [inline templates/legal/terms-and-policies/privacy-policy/third-parties.html]: This is breaking CI

File: templates/legal/terms-and-policies/privacy-policy/third-parties.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h1>Third-party privacy policies</h1>
       <li class="p-list__item"><a href="http://cleardarksky.com/csk/admin.html#Privacy">Clear Sky Chart</a></li>
       <li class="p-list__item"><a href="http://www.colourlovers.com/privacy">COLOURlovers</a></li>
       <li class="p-list__item"><a href="http://www.dailymotion.com/us/legal/privacy">Dailymotion</a></li>
-      <li class="p-list__item"><a href="http://about.deviantart.com/policy/privacy/">deviantArt</a></li>
+      <li class="p-list__item"><a href="http:/about.deviantart.com/policy/privacy/">deviantArt</a></li>
```
Comment (Caleb-Ellis) [inline templates/legal/terms-and-policies/privacy-policy/third-parties.html]: And so is this

**Pull request merged**